driver-async/src/main/com/mongodb/async/client/gridfs/GridFSUploadStreamImpl.java

```diff
@@ -179,7 +172,7 @@ public void onResult(final Void result, final Throwable t) {
                     errHandlingCallback.onResult(null, t);
                 } else {
                     GridFSFile gridFSFile = new GridFSFile(fileId, filename, lengthInBytes, chunkSizeBytes, new Date(),
-                            getMD5Digest(), metadata);
+                            null, metadata);
```

Comment:
What is the plan for MD5 in GridFSFile? Can it be removed also? I would be available in the `extraElements` field.

Reply:
MD5 will be fully eradicated in later patches